sonar-php-plugin/src/test/java/org/sonar/plugins/php/PhpPluginTest.java

```diff
@@ -22,14 +22,18 @@
  */
 package org.sonar.plugins.php;
 
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
 import org.junit.Test;
 import org.sonar.api.Plugin;
 import org.sonar.api.SonarQubeSide;
 import org.sonar.api.SonarRuntime;
+import org.sonar.api.config.PropertyDefinition;
 import org.sonar.api.internal.SonarRuntimeImpl;
 import org.sonar.api.utils.Version;
 
-import static org.fest.assertions.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
@inverno let's have a convention on this as well:) I use `fest` (for no reason in fact)